docs/How-We-Work/team-charter/definition-of-done

```diff
@@ -0,0 +1,15 @@
+# **Definition of Done**
+In order to define an issue as done, it needs to have passed a Raft Review and the [QASP Review](https://github.com/HHS/TANF-app/blob/main/docs/How-We-Work/Quality-Expectations.md) and follows [our branching strategy](https://github.com/HHS/TANF-app/blob/main/docs/How-We-Work/Branching-Strategy.md) which results in the code being available in the HHS/TANF-app repo.
+
+The following checklist needs to be completed:
```

Comment:
Should we link to this checklist rather than writing it out? If we write it out we'll need to make future updates to the checklist in two places instead of one.

Reply:
yes... I'm still trying to figure out where everything is, but I'll keep digging to find the document and do that instead. Thanks!

Reply:
Of course! Check out `.github/pull_request_template.md`